Index: iw/recipe/pound/pound.cfg.tpl
===================================================================
--- iw/recipe/pound/pound.cfg.tpl	(révision 8679)
+++ iw/recipe/pound/pound.cfg.tpl	(copie de travail)
@@ -32,6 +32,10 @@
 # How long should Pound continue to answer existing connections after a receiving and INT or HUP signal
 Grace $grace
 
+
+# Socket
+Control "$socket"
+
 # listen, redirect and ... to:
 #for $balancer in $balancers
 # balancer for $balancer.name
Index: iw/recipe/pound/config.py
===================================================================
--- iw/recipe/pound/config.py	(révision 8679)
+++ iw/recipe/pound/config.py	(copie de travail)
@@ -43,6 +43,7 @@
 
     def __init__(self, buildout, name, options):
         self.buildout = buildout
+        self.name = name
         location = options.get(
             'location', buildout['buildout']['parts-directory'])
         options['location'] = os.path.join(location, name)
@@ -92,6 +93,15 @@
 
         tpl.owner = self.buildoptions.get('owner', utils.get_sysuser())
         tpl.group = self.buildoptions.get('group', tpl.owner)
+        
+        tpl.socket = self.buildoptions.get(
+            'socket', 
+            os.path.join(
+                self.buildout['buildout']['directory'], 
+                'var', 
+                'pound.sock'
+            )
+        )
 
 
 
@@ -170,13 +180,15 @@
 
         pid = os.path.join(var_dir, 'pound.pid')
         scripts = []
+        
+        scripts_names = {'poundctl': self.name}
         for x in ('poundctl', 'poundrun'):
             ctl = Template(open(os.path.join(curdir, x+'.tpl')).read())
             ctl.poundbin = self.options['executable']
             ctl.poundcfg = filename
             ctl.poundpid = pid
             bin_dir = self.buildout['buildout']['bin-directory']
-            script_name = os.path.join(bin_dir, x)
+            script_name = os.path.join(bin_dir, scripts_names.get(x, x))
             f = open(script_name, 'w')
             try:
                 print >>f, ctl
Index: setup.py
===================================================================
--- setup.py	(révision 8679)
+++ setup.py	(copie de travail)
@@ -21,7 +21,7 @@
 import os
 from setuptools import setup, find_packages
 
-version = '0.5.0'
+version = '0.5.1'
 
 README = os.path.join(os.path.dirname(__file__),
               'iw',
Index: CONTRIBUTORS.txt
===================================================================
--- CONTRIBUTORS.txt	(révision 8679)
+++ CONTRIBUTORS.txt	(copie de travail)
@@ -1,4 +1,4 @@
 Ingeniweb Internal sprint 2008
 Tarek Ziade
 Youenn Boussard
-
+Makina Corpus <freesoftware@makina-corpus.com>
